src/handlers/miner_hbbft_handler.erl

```diff
@@ -463,7 +462,7 @@ deserialize(#{sk := SKSer,
                   case StateMap of
                       #{K := V} when V /= undefined andalso
                                      V /= Bundef ->
-                          binary_to_term(V);
+                          V;
                       _ when K == sig_phase ->
                           sig;
                       _ when K == seen ->
```

Comment:
Since we're ignoring these fields in `deserialize` do we need to serialize them in the first place? Skipping these fields in `serialize` will reduce the number of field-by-field compares in `relcast`.

Reply:
I think its populating the default values if they're not defined.

Reply:
OK, I get it now. The first case clause checks for `undefined` and falls through to the others if it is.